src/modules/About/About.css

```diff
@@ -20,18 +20,18 @@
 }
 
 .flex-row-container {
-    display: flex;
-    flex-wrap: wrap;
-    align-items: center;
-    justify-content: center;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
 }
```

Comment:
Should add a line space for better readability and to make it more consistent with other elements.
Other than that, the About page looks good.

Reply:
Follow above advice.

Reply:
Done